cmake/tribits/ctest_driver/TribitsCTestDriverCoreHelpers.cmake

```diff
@@ -1368,10 +1372,6 @@ MACRO(TRIBITS_CTEST_ALL_AT_ONCE)
     ELSE()
       MESSAGE("Configure PASSED!")
       SET(AAO_CONFIGURE_PASSED TRUE)
-      # Load target properties and test keywords
-      CTEST_READ_CUSTOM_FILES(BUILD "${CTEST_BINARY_DIRECTORY}")
-      # Overridde from this file!
-      INCLUDE("${TRIBITS_PROJECT_ROOT}/CTestConfig.cmake")
```

Comment:
Just making sure these lines were supposed to be removed.  I couldn't figure out a reason just looking at them.